.gitignore

```diff
@@ -3,3 +3,4 @@
 lib/
 node_modules
 usage.txt
+.idea/
```

Comment:
Can you please remove this? I strongly agree with @jbpros and would prefer to take that opportunity to teach people to use their local global gitignore.